### PR TITLE
Replace `cmdtools` module to `requests`

### LINKED ISSUE #37: Remove `cmdtools`
We decided to remove the `cmdtools` module as we are not going to use the artiq client directly.
The [snu-quiqcl/artiq-proxy](https://github.com/snu-quiqcl/artiq-proxy) repository will provide an artiq-independent interface for queries, as mentioned in #22.

### PR DESCRIPTION
**Pull request by BECATRUE**
This will close #37.

I removed the `cmdtools` module and use the `requests` module to request to the proxy server.
Even if the `requests` module is external, in the official [documentation](https://docs.python.org/3/library/http.client.html), it recommends this module, so I chose it.

Also, as snu-quiqcl/artiq-proxy#14, we added the `repository_path` to the proxy server internally, so I modified it.

Now, the unittest would be failed. I will modify it in #38.

### CODE REVIEW

File: iquip/apps/explorer.py [hunk 1]
```diff
@@ -69,34 +68,34 @@ def __init__(
...
-        experimentList = experimentList.split("\n")[:-1]  # The last one is always an empty string.
+        try:
+            response = requests.get("http://127.0.0.1:8000/ls/",
+                                    params={"directory": self.path},
+                                    timeout=10)
+            response.raise_for_status()
+            experimentList = response.json()
+        except requests.exceptions.RequestException:
+            pass
         self.fetched.emit(experimentList, self.widget)
```
Comment (kangz12345) [inline iquip/apps/explorer.py:86]: This code is executed regardless if the get request succeeds.

File: iquip/apps/explorer.py [hunk 2]
```diff
@@ -69,34 +68,34 @@ def __init__(
...
-        experimentList = cmdtools.run_command(f"artiq_client ls {self.path}").stdout
-        experimentList = experimentList.split("\n")[:-1]  # The last one is always an empty string.
+        try:
+            response = requests.get("http://127.0.0.1:8000/ls/",
+                                    params={"directory": self.path},
+                                    timeout=10)
+            response.raise_for_status()
+            experimentList = response.json()
+        except requests.exceptions.RequestException:
+            pass
```
Comment (kangz12345) [inline iquip/apps/explorer.py]: Even though we are not logging by now, but let's temporarily print something about the exception information.

**Review by kangz12345 — CHANGES_REQUESTED** [review]

File: iquip/apps/explorer.py (hunk 1 above)
Comment (BECATRUE) [reply iquip/apps/explorer.py:86]: Oh, you're right! I will add a printed message and return here in the handling part.

**Review by kangz12345 — APPROVED** [review]
LGTM

**Pull request merged**